server/api/internal/infrastructure/websocket/client.go

```
@@ -151,37 +151,49 @@
-	defer C.free(unsafe.Pointer(cConfigJSON))
+	req, err := http.NewRequestWithContext(ctx, http.MethodPost, url, bytes.NewBuffer(jsonData))
+	if err != nil {
+		return nil, fmt.Errorf("failed to create request: %w", err)
+	}
+	req.Header.Set("Content-Type", "application/json")
+
+	resp, err := c.client.Do(req)
+	if err != nil {
+		return nil, fmt.Errorf("failed to send request: %w", err)
+	}
+	defer resp.Body.Close()
```

golangci-lint (1.62.2)

Error return value of `resp.Body.Close` is not checked

(errcheck)